.npmrc

```diff
@@ -0,0 +1 @@
+@NAMESPACE:registry=https://npm.pkg.github.com
```

Comment:
Why are you wanting to change the registry? We upload all of our packages to npm https://npmjs.com/org/pretendonetwork

Reply:
that was an accident, was meant to go to my fork